tests/unit/metrics_test.cc

```diff
@@ -319,3 +320,57 @@ SEASTAR_THREAD_TEST_CASE(test_relabel_enable_disable_skip_when_empty) {
     }), 0);
     sm::set_relabel_configs({}).get();
 }
+
+SEASTAR_THREAD_TEST_CASE(test_estimated_histogram) {
+    using namespace seastar::metrics;
+    using namespace std::chrono_literals;
+    time_estimated_histogram histogram1;
+    time_estimated_histogram histogram2;
+    time_estimated_histogram::duration min = std::chrono::microseconds(512);
```

Comment:
nit: magic number. It should have a comment, like:
```
// The lower bound of time_estimated_histogram is 512 us.
time_estimated_histogram::duration min = std::chrono::microseconds(512);
```

Reply:
It's a test. Those are arbitrary numbers. I chose the same numbers I used for time_estimated_histogram, but it could have been anything (as long as it's  power of 2)

Reply:
> Those are arbitrary numbers. I chose the same numbers I used for time_estimated_histogram, but it could have been anything (as long as it's power of 2)

What do you mean? If you changed the 512 here to any other number, the test would fail. Likewise, if any constant in time_estimated_histogram changed, the test would fail.